disko.py

```diff
@@ -1,4 +1,14 @@
-from src.disko.image_collector import ImageCollector
+from src.mvc.view import ImageRegistryManager
```

Comment:
let's leave the structure with src.disko.xx.xxx as it's important for future integration. @Daniel-Shwartzman 
also change the mvc folder to something else.